### ISSUE TITLE
Memory safe assembly blocks cause parse error

### ISSUE THREAD

**Issue by patrickd-** [solidity-parser/parser#67]
Changing

```diff
-        assembly {
+        assembly ("memory-safe") {
```

as per https://docs.soliditylang.org/en/v0.8.13/assembly.html#memory-safety

will cause a parser error (in solhin):

```
  195:17  error  Parse error: extraneous input '(' expecting {'{', StringLiteralFragment}
  195:31  error  Parse error: extraneous input ')' expecting '{'
```

**Comment by fvictorio** [comment]
Hi @patrickd-, I'm pretty sure this is fixed in the latest version of the parser, so this should be an issue in the solhint repo about updating their dependency.

**Comment by fvictorio** [comment]
Oh, yes, sorry, that functionality is in the `0.14.2-beta.1` version.

**Comment by fvictorio** [comment]
I just published `0.14.12` as the latest version. Please let me know if that still doesn't work.

**Comment by patrickd-** [comment]
That works, thanks!
